pkgdb/tests/pathological.bats

```diff
@@ -10,275 +10,254 @@
+  assert_success
   run sqlite3 "$DBPATH" "SELECT license FROM Packages  \
-    WHERE attrName = 'pkg2' LIMIT 1";
-  refute_output --regexp '.';
+    WHERE attrName = 'pkg2' LIMIT 1"
+  refute_output --regexp '.'
 }
 
-
 # ---------------------------------------------------------------------------- #
 
 @test "pkg2 'unfree' set with bad license" {
-  run "$PKGDB_BIN" scrape --database "$DBPATH" "$TEST_HARNESS_FLAKE"  \
-                    packages "$NIX_SYSTEM";
-  assert_success;
+  run "$PKGDB_BIN" scrape --database "$DBPATH" "$TEST_HARNESS_FLAKE" \
+                    packages "$NIX_SYSTEM"
+  assert_success
   run sqlite3 "$DBPATH" "SELECT unfree FROM Packages  \
-    WHERE attrName = 'pkg2' LIMIT 1";
-  assert_output '0';
+    WHERE attrName = 'pkg2' LIMIT 1"
+  assert_output '0'
 }
 
-
 # ---------------------------------------------------------------------------- #
 
 @test "pkg3 'name' constructed" {
-  run "$PKGDB_BIN" scrape --database "$DBPATH" "$TEST_HARNESS_FLAKE"  \
-                    packages "$NIX_SYSTEM";
-  assert_success;
+  run "$PKGDB_BIN" scrape --database "$DBPATH" "$TEST_HARNESS_FLAKE" \
+                    packages "$NIX_SYSTEM"
+  assert_success
   run sqlite3 "$DBPATH" "SELECT name FROM Packages  \
-    WHERE name = 'pkg-2023-08-09' LIMIT 1";
-  assert_output 'pkg-2023-08-09';
+    WHERE name = 'pkg-2023-08-09' LIMIT 1"
+  assert_output 'pkg-2023-08-09'
 }
 
-
 # ---------------------------------------------------------------------------- #
 
 @test "pkg3 has no 'semver' attr" {
-  run "$PKGDB_BIN" scrape --database "$DBPATH" "$TEST_HARNESS_FLAKE"  \
-                    packages "$NIX_SYSTEM";
-  assert_success;
+  run "$PKGDB_BIN" scrape --database "$DBPATH" "$TEST_HARNESS_FLAKE" \
+                    packages "$NIX_SYSTEM"
+  assert_success
   run sqlite3 "$DBPATH" "SELECT semver FROM Packages  \
-    WHERE attrName = 'pkg3' LIMIT 1";
-  refute_output --regexp '.';
+    WHERE attrName = 'pkg3' LIMIT 1"
+  refute_output --regexp '.'
 }
 
-
 # ---------------------------------------------------------------------------- #
 
 @test "pkg4 'name' == 'pname'" {
-  run "$PKGDB_BIN" scrape --database "$DBPATH" "$TEST_HARNESS_FLAKE"  \
-                    packages "$NIX_SYSTEM";
-  assert_success;
+  run "$PKGDB_BIN" scrape --database "$DBPATH" "$TEST_HARNESS_FLAKE" \
+                    packages "$NIX_SYSTEM"
+  assert_success
   run sqlite3 "$DBPATH" "SELECT pname FROM Packages  \
-    WHERE attrName = 'pkg4' LIMIT 1";
-  assert_output 'pkg';
+    WHERE attrName = 'pkg4' LIMIT 1"
+  assert_output 'pkg'
 }
 
-
 # ---------------------------------------------------------------------------- #
 
 @test "pkg4 has no 'broken' attr" {
-  run "$PKGDB_BIN" scrape --database "$DBPATH" "$TEST_HARNESS_FLAKE"  \
-                    packages "$NIX_SYSTEM";
-  assert_success;
+  run "$PKGDB_BIN" scrape --database "$DBPATH" "$TEST_HARNESS_FLAKE" \
+                    packages "$NIX_SYSTEM"
```

Comment:
Question: Is there a rule for indentation after a line continuation that we can use?

Reply:
the rule was `keep_padding`
i.e. keep manual indentiation in continued lines like this